tests/avifgridapitest.c

```diff
@@ -3,23 +3,41 @@
 
 #include "avif/avif.h"
 
+#include <assert.h>
 #include <inttypes.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 
 //------------------------------------------------------------------------------
 
+// Fills a plane with a repeating gradient.
+static void fillPlane(int width, int height, int depth, uint8_t * row, uint32_t rowBytes)
+{
+    assert(depth == 8 || depth == 10 || depth == 12); // Values allowed by AV1.
+    const int maxValuePlusOne = 1 << depth;
+    for (int y = 0; y < height; ++y) {
+        if (depth == 8) {
+            memset(row, y % maxValuePlusOne, width);
+        } else {
+            for (int x = 0; x < width; ++x) {
+                ((uint16_t *)row)[x] = (uint16_t)(y % maxValuePlusOne);
+            }
+        }
+        row += rowBytes;
+    }
+}
+
 // Creates an image where the pixel values are defined but do not matter.
 // Returns false in case of memory failure.
-static avifBool createImage(int width, int height, int depth, avifPixelFormat yuvFormat, avifImage ** image)
+static avifBool createImage(int width, int height, int depth, avifPixelFormat yuvFormat, int createAlpha, avifImage ** image)
```

Comment:
Please use the `avifBool` type for the `createAlpha` parameter. Search for "int createAlpha" in this file and change all occurrences.